docs/event-schemas/wfm/WorkflowRunStateChange.json

```diff
@@ -0,0 +1,104 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "WorkflowRunStateChange"
+  },
+  "paths": {},
+  "components": {
+    "schemas": {
+      "AWSEvent": {
+        "type": "object",
+        "required": [
+          "detail-type",
+          "detail",
+          "id",
+          "source",
+          "time"
+        ],
+        "properties": {
+          "detail": {
+            "$ref": "#/components/schemas/WorkflowRunStateChange"
+          },
+          "account": {
+            "type": "string"
+          },
+          "detail-type": {
+            "type": "string"
+          },
+          "id": {
+            "type": "string"
+          },
+          "region": {
+            "type": "string"
+          },
+          "resources": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "source": {
+            "type": "string"
+          },
+          "time": {
+            "type": "string",
+            "format": "date-time"
+          },
+          "version": {
+            "type": "string"
+          }
+        }
+      },
+      "WorkflowRunStateChange": {
+        "type": "object",
+        "required": [
+          "portalRunId",
+          "timestamp",
+          "status",
+          "workflowType",
+          "workflowVersion",
+          "payload",
+          "serviceVersion"
```

Comment:
Do we need a `service` or is the `source` sufficient?

Reply:
What's the difference between service and source?

Reply:
I think Flo is referring to `serviceVersion`

Reply:
> The service differentiator marker on events is the `source` property of the event envelope.

Reply:
We discussed about pros/cons of opaqueness of Payload object. And, tried tinker with alternate approaches such as fixing the schemas upfront.

For now, we concluded we shall keep it as Payload object.

The `serviceVersion` will be refactored to `version` property of the `Payload` definition as the required field; along with payload `refId`.

Reply:
Updated with https://github.com/umccr/orcabus/commit/1e903c67c1eb0fe8ffa949171a058c93c835c4d7